docs/migrating_2.rst

```diff
@@ -29,6 +29,8 @@ Overview
 - ``missing_perms`` attributes and arguments are renamed to ``missing_permissions``.
 - Many method arguments now reject ``None``.
 - Many arguments are now specified as positional-only or keyword-only; e.g. :func:`oauth_url` now takes keyword-only arguments, and methods starting with ``get_`` or ``fetch_`` take positional-only arguments.
+- ``AppInfo.summary`` has been removed.
+- Due to a recent API change, existing attachments must be specified when editing messages with new attachments. Any attachments not specified will be removed and replaced with the specified list.
```

Comment:
This should not just be in the overview but with some more text below aswell

Reply:
?

Reply:
can you consider help me write it since i have no idea what to write

Reply:
``AppInfo.summary`` has been removed. - not removed, just will always be an empty string
keep that longer description to outside of the overview and in the bigger descriptions down below